### ISSUE TITLE
[HOLD for payment 2025-01-15] Android - Chat - Emoji picker is jumpy when opened from context menu

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#54453]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.77-6
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** applausetester+kh1311020@applause.expensifail.com
**Issue reported by:** Applause - Internal Team
**Component:** Chat Report View

## Action Performed:
1. Launch ND or hybrid app
2. Open DM
3. Send a message to the DM if there is no message
4. Tap on the composer if keyboard is not up
5. Long tap on the message while keyboard is up
6. Tap Add reaction button
7. Might need to repeat Step 4 to 6 if the emoji picker is not jumpy

## Expected Result:
Emoji picker will not be jumpy when opened from context menu when keyboard is opened

## Actual Result:
Emoji picker is jumpy when opened from context menu when keyboard is opened

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Standalone
- [x] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/52aaa9ac-0ccb-4f97-a35c-eccbb1bbf6c5

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by lanitochka17** [comment]
Production:

Uploading Screenrecorder-2024-12-22-20-33-16-287.mp4…

**Comment by techievivek** [comment]
Doesn't seem to be a backend bug so removing the Deployblocker label from it.

**Comment by arosiclair** [comment]
Attempting to reproduce this in dev. I'll see if reverting a couple PRs fixes the issue.

**Comment by arosiclair** [comment]
Reverting https://github.com/Expensify/App/pull/54187 seems to fix it but I'm not 100% sure. @kirillzyusko can you take a look at this?

In the meantime I'm going to demote this since it's minor.

**Comment by kirillzyusko** [comment]
@arosiclair yes, sure, feel free to assign me on this issue 👀 Will be happy to help here!

**Comment by kirillzyusko** [comment]
@arosiclair it's not a deploy blocker, right?

The thing is why `EmojiPicker` is jumpy is because we instantly change padding. And it always was looking weird. I believe with the help of keyboard-controller I can achieve smooth animations and re-work the code.

just want to understand whether it's a `DeployBlocker` or not (whether do we need to revert a PR and prepare a new PR that will fix prev and that issue) 🤔

**Comment by arosiclair** [comment]
> @arosiclair it's not a deploy blocker, right?

No it's not. I think it's relatively minor so I demoted this issue.

**Comment by arosiclair** [comment]
@parasharrajat was the C+ for this

**Comment by kirillzyusko** [comment]
@danstepanov is working on this to fix 👀

**Comment by mallenexpensify** [comment]
Contributor+: @parasharrajat due $250 via NewDot 

Test Case
- https://github.com/Expensify/Expensify/issues/462308

**Comment by JmillsExpensify** [comment]
$250 approved for @parasharrajat
